app/src/main/res/values/strings.xml

```diff
@@ -105,11 +105,70 @@
     <string name="firstrun_footer">To be clear, sites and service you use can still know where you\'ve been.</string>
 
     <string name="your_rights">Your Rights</string>
+
     <!-- This string is shown when the user has clicked a link that needs to be opened in a different app.
          Argument 2 is the external app name, argument 1 is our name (i.e. Firefox Focus, or Firefox Klar). -->
     <string name="external_app_prompt">This link will open in %2$s. Are you sure you want to exit %1$s?</string>
     <string name="external_app_prompt_no_app_title">No app to handle link</string>
     <string name="external_app_prompt_no_app">None of the apps on your device are able to open this link. Do you want to exit %1$s and open %2$s to find a suitable app?</string>
     <!-- This label is shown above a list of apps that can be used to open a given link -->
     <string name="external_multiple_apps_matched_exit">Exit Private Browsing?</string>
+
+    <!-- The page html title (i.e. the <title> tag content) -->
+    <string name="errorpage_title">Problem loading page</string>
+    <string name="errorpage_refresh">Try Again</string>
+
+    <string name="error_connectionfailure_title">Unable to connect</string>
+    <string name="error_connectionfailure_message">
+    <![CDATA[
+      <ul>
+        <li>The site could be temporarily unavailable or too busy. Try again in a few moments.</li>
+        <li>If you are unable to load any pages, check your mobile device’s data or Wi-Fi connection.</li>
+      </ul>
+    ]]>
+    </string>
+
+    <string name="error_connect_title">@string/error_connectionfailure_title</string>
+    <string name="error_connect_message">@string/error_connectionfailure_message</string>
+
+    <string name="error_timeout_title">The connection timed out</string>
+    <string name="error_timeout_message">@string/error_connectionfailure_message</string>
+
+    <string name="error_hostLookup_title">Server not found</string>
+    <string name="error_hostLookup_message"><![CDATA[
+      <ul>
+        <li>Check the address for typing errors such as
+          <strong>ww</strong>.example.com instead of
+          <strong>www</strong>.example.com</li>
+        <li>If you are unable to load any pages, check your device’s data or Wi-Fi connection.</li>
+      </ul>
+    ]]></string>
+
+
+    <string name="error_malformedURI_title">The address isn’t valid</string>
+    <string name="error_malformedURI_message"><![CDATA[
+<ul>
+  <li>Web addresses are usually written like  <strong>http://www.example.com/</strong></li>
+  <li>Make sure that you’re using forward slashes (i.e. <strong>/</strong>).</li>
+</ul>
+    ]]></string>
+
+    <string name="error_redirectLoop_title">The page isn’t redirecting properly</string>
+    <string name="error_redirectLoop_message"><![CDATA[<ul><li>This problem can sometimes be caused by disabling or refusing to accept cookies.</li></ul>]]></string>
```

Comment:
Maybe worth showing these on multiple lines, structuring the code like the others? In case, same for `error_unsupportedprotocol_message`